tests/books/test_books.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from tests.factories import BookFactory
+
+BASE = "/api/v1/books/"
+
+
+new_book_payload = {
+        "title": "New",
+        "author": "A",
+        "cover": "SOFT",
+        "inventory": 2,
+        "daily_fee": "1.20",
+    }
```

Comment:
obiekty mutowalne powinny być fixturami. Gwarantuje to, ze gdyby jeden test podmienił jakąś wartość, to w ramach kolejnego testu będziemy mieli `świeży` słownik